HISTORY.md

```diff
@@ -21,8 +21,22 @@
   * Fix floating-point accuracy issue for decision trees that sometimes caused
     crashes (#3595).
 
+  * Use templates metaprog to distinguish between a matrix and a cube type
+    (#3602), (#3585).
+
+  * Use `MatType` instead of `arma::Mat<eT>`, (#3567), (#3607), (#3608),
+    (#3609), (#3568).
+
+  * Remove `arma` from `arma::function`, (#3619), (#3617) (#3610), (#3643), (#3600),
+    (#3605), (#3629).
+
+  * Change `arma::conv_to` to `ConvTo` using a local shim (#3614).
+
+  * Fix a bug for the stddev and mean in `RandNormal()` #(3651).
+
   * Allow PCA to take different matrix types (#3677).
 
+>>>>>>> master
```

Comment:
Oops, still a merge artifact here?

Reply:
Oops, this indeed is a merge artifact,
Will fix that shortly, and include the above message